app/models/app.js

```diff
@@ -17,6 +17,7 @@ export default DS.Model.extend(ProvisionableMixin, {
   lastDeployOperation: DS.belongsTo('operation', {async:true}),
   currentImage: DS.belongsTo('image', {async:true}),
 
+  reloadWhileProvisioning: true,
```

Comment:
I fixed the issue with `STATUSES`; I added the enumeration in a separate file and imported it in each file that requires it.